### PR TITLE
liblouis-tables 2.6.2

### PR DESCRIPTION
**Pull request by bertfrees**

### CODE REVIEW

File: pipeline-braille-utils/liblouis/liblouis-tables/pom.xml [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
   <packaging>bundle</packaging>
 
   <name>DAISY Pipeline 2 Braille Module :: liblouis-tables</name>
 
   <properties>
-    <liblouis.version>2.5.4</liblouis.version>
-    <liblouis.tarball.url>http://www.liblouis.org/downloads/liblouis-2.5.4.tar.gz</liblouis.tarball.url>
-    <liblouis.tarball.sha1>d2c6727973760560235daab7a8830f19b31adb49</liblouis.tarball.sha1>
+    <liblouis.version>2.6.2</liblouis.version>
+    <liblouis.tarball.url>https://github.com/liblouis/liblouis/releases/download/v2.6.2/liblouis-2.6.2.tar.gz</liblouis.tarball.url>
```
Comment (bertfrees) [inline pipeline-braille-utils/liblouis/liblouis-tables/pom.xml]: This build doesn't include the new Finnish table yet. @egli have you uploaded the right v2.6.2 (i.e. after the last minute change)?

**Comment by egli** [comment]
I did. But I think there is a bug in that the fi.utb table is not included when running make dist. Just pushed a fix.

Whenever a new table is added in tables it needs to be added to Makefile.am. Otherwise it will not be in the tar ball.

**Comment by bertfrees** [comment]
Ah really? Then I'll have to check out the source tarball instead.

**Comment by egli** [comment]
Or just build another tar ball using the 45f1430d8839dfe522e943ee842a723db0887957 checkout

**Comment by bertfrees** [comment]
Rebased onto https://github.com/snaekobbi/pipeline-mod-braille/pull/35

**Comment by bertfrees** [comment]
Rebased onto c2d72ef6b91b0c6ca8449a65f0b5d476c6a3694b and merged into master.

**Pull request closed without merging**